swagger.json

```diff
@@ -0,0 +1,4297 @@
+{
```

Comment:
There is already an API spec in this repo. Can we update that one instead of adding a second?

Reply:
Removed dup swagger spec.